lib/components/Keyboard/KeyboardTrackingView/keyboardTrackingView.api.json

```diff
@@ -21,63 +21,54 @@
     {
       "name": "scrollBehavior",
       "type": "number",
-      "description": "The scrolling behavior (use KeyboardTrackingView.scrollBehaviors.NONE | SCROLL_TO_BOTTOM_INVERTED_ONLY | FIXED_OFFSET)",
-      "note": "iOS only"
```

Comment:
You've removed these because they are all for iOS?

Reply:
Yes. See component description